controllers/telegram_handlers.py

```diff
@@ -272,21 +272,29 @@ async def get_exam_schedule_handler(update: Update, context: ContextTypes.DEFAUL
         )
 
         response = await get_exam_schedule(user_id)
-        if response is None:
+
+        if response == "not_logged":
+            await context.bot.send_message(
+            chat_id=user_id,
+            text="There was an error, maybe you are not logged in. Use /login {amizone_id} {password} to login.",
+            )
+            return
+        elif response is None:
             # ! Need better exception handling
             await context.bot.send_message(
                 chat_id=user_id,
-                text="There was an error, maybe you are not logged in. Use /login {amizone_id} {password} to login.",
+                text="No Exams! Chill......🤓",
```

Comment:
Since this message from the bot is the end of a conversation, we have to return ``InlineKeyboardMarkup(BUTTON_MARKUP)``